src/ipc-dbus/DBusIPCAPI_v1.cpp

```diff
@@ -1340,13 +1355,15 @@ DBusIPCAPI_v1::interface_route_remove_handler(
 	struct in6_addr address = {};
 	bool did_succeed(false);
 
-	did_succeed = dbus_message_get_args(
-		message, NULL,
-		DBUS_TYPE_ARRAY, DBUS_TYPE_BYTE, &route_prefix, &prefix_len_in_bytes,
-		DBUS_TYPE_UINT16, &domain_id,
-		DBUS_TYPE_BYTE, &prefix_len_in_bits,
-		DBUS_TYPE_INVALID
-	);
+	if (!did_succeed) {
```

Comment:
why add the `if` condition which would always be true here?

Reply:
Thanks. You are correct. We don't need it. 
Was originally planning to add stable flag to "remove" as well, but we don't need it.